PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Windows;
+
+using PowerPointLabs.Utils;
+
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.PasteLab
+{
+    public class PasteLabMain
```

Comment:
The idea is the log might be useful if a user reports an error and we want to know what happened. The user might say something like 'PasteLab stopped working after pasting a few items' and then we want to know exactly what sequence of actions led to the error.